views/index.html

```diff
@@ -27,9 +27,131 @@
+        </div>
+      </article>
+      <article class="project-card">
+        <img src="../assets/images/projects/project-3.png" alt="project-3">
+        <div class="card-body">
+          <h2 class="card-header">Project name goes here</h2>
+          <ul class="languages">
+            <li class="lang">HTML/CSS</li>
+            <li class="lang">Ruby on Rails</li>
+            <li class="lang">Javascript</li>
+          </ul>
+          <a href="#" class="project-link">See this project<i class="fas fa-arrow-right"></i></a>
+        </div>
+      </article>
+      <article class="project-card">
+        <img src="../assets/images/projects/project-4.png" alt="project-4">
+        <div class="card-body">
+          <h2 class="card-header">Project name goes here</h2>
+          <ul class="languages">
+            <li class="lang">HTML/CSS</li>
+            <li class="lang">Ruby on Rails</li>
+            <li class="lang">Javascript</li>
+          </ul>
+          <a href="#" class="project-link">See this project<i class="fas fa-arrow-right"></i></a>
+        </div>
+      </article>
+      <article class="project-card">
+        <img src="../assets/images/projects/project-5.png" alt="project-5">
+        <div class="card-body">
+          <h2 class="card-header">Project name goes here</h2>
+          <ul class="languages">
+            <li class="lang">HTML/CSS</li>
+            <li class="lang">Ruby on Rails</li>
+            <li class="lang">Javascript</li>
+          </ul>
+          <a href="#" class="project-link">See this project<i class="fas fa-arrow-right"></i></a>
+        </div>
+      </article>
+      <article class="project-card">
+        <img src="../assets/images/projects/project-6.png" alt="project-6">
+        <div class="card-body">
+          <h2 class="card-header">Project name goes here</h2>
+          <ul class="languages">
+            <li class="lang">HTML/CSS</li>
+            <li class="lang">Ruby on Rails</li>
+            <li class="lang">Javascript</li>
+          </ul>
+          <a href="#" class="project-link">See this project<i class="fas fa-arrow-right"></i></a>
+        </div>
+      </article>
+    </section>
+    <section class="about-section margin-x">
+      <article class="about-header">
+        <h1 class="display-1">About</h1>
+        <p class="par">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+          labore
+          et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+          ea
+          eiusmod</p>
+        <button class="btn" type="button"><span>Get mt CV<i class="fas fa-download"></i></span></button>
+      </article>
+      <article class="squares-wrapper">
+        <div class="square square-1">
+          <h2>Languages</h2>
+          <ul>
+            <li><span>JavaScript (ES5/ES6)</span></li>
```

Comment:
## [OPTIONAL]
- [ ] kindly compare your design on the right side and the one from the template on the left side of the image down here and then see how you can improve it. 

![Screenshot from 2021-11-26 00-25-32](https://user-images.githubusercontent.com/65068771/143504787-aeaac5d9-3b0e-4f0f-8c27-e7445aafbbb8.png)